content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -43,13 +43,48 @@ By default, the `/query` compatibility endpoint returns results in **JSON**.
 To return results as **CSV**, include the `Accept: application/csv` header.
 
 ## Database and retention policy mapping
+
 InfluxDB 2.0 combines the 1.x concept of [databases](/influxdb/v1.8/concepts/glossary/#database)
 and [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 into [buckets](/influxdb/v2.0/reference/glossary/#bucket).
 To support InfluxDB 1.x query and write patterns in InfluxDB 2.0, databases and retention
 policies are mapped to buckets using the **database and retention policy (DBRP) mapping service**.
 _See [DBRP mapping](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) for more information._
```

Comment:
Perhaps we should explicitly say InfluxQL looks for a database and RP, somewhere in intro:

InfluxQL looks for a [database](/influxdb/v1.8/concepts/glossary/#database) and [retention policy](/influxdb/v1.8/concepts/glossary/#retention-policy-rp) (elements in the FROM clause) to query from. 
To query a 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket) using InfluxQL, you must first map the bucket  to a database and retention policy.

Reply:
We might add a note...in case they've already written data:

Note: If you use the 1.x compatibility API to write data into InfluxDB 2.0, the database and retention policy included as query parameters were mapped to a bucket. For more detail, see [Database and ret.....](https://docs.influxdata.com/influxdb/v2.0/reference/api/influxdb-1x/dbrp/).